components/brave_wallet/browser/internal/cardano_tx_decoder.cc

```diff
@@ -20,10 +22,27 @@ namespace {
 // https://cips.cardano.org/cip/CIP-0009#updatable-protocol-parameters
 constexpr size_t kMaxCardanoTransactionSize = 16 * 1024;  // 16384
 
+rust::Vec<uint8_t> ToRust(base::span<const uint8_t> data) {
```

Comment:
Why are we using rust::Vec instead of rust::Slice? There is base::SpanToRustSlice for the latter case